tools/docker-compose/README.md

```diff
@@ -190,6 +190,76 @@ awx_1        |   Applying auth.0001_initial... OK
 ...
 ```
 
+##### Keycloak Integration
```

Comment:
I'd prefer to put this under the "Extras" section, similar to the "Start with Minikube" section.